packages/react/src/React.js

```diff
@@ -35,53 +36,49 @@ import {
 } from './ReactElementValidator';
 import ReactDebugCurrentFrame from './ReactDebugCurrentFrame';
 
-const React = {
-  Children: {
-    map,
-    forEach,
-    count,
-    toArray,
-    only,
-  },
+export const Children = {
+  map,
+  forEach,
+  count,
+  toArray,
+  only,
+};
```

Comment:
Why not `import * as Children from './ReactChildren'; export { Children };`? If `ReactChildren` `export`s only the public API, this would allow for tree shaking.

Reply:
Afaik rollup is able to treeshake object literals. Not sure webpack able to handle any of cases.

Reply:
Only namespace reexports are tree-shakeable by webpack; by storing the result in an actual object you prevent webpack's tree shaking.

Reply:
@Kovensky not sure if you statements agrees with me or not, just to test this once again - even if only for myself - I've prepared a quick test to see if they are treeshakeable by webpack or not and according to my results they are not
https://github.com/Andarist/webpack-reexported-namespace-test/blob/cf141855cf341a9cf0ea673d760546f77438e6ca/dist/main.js#L47-L54